creatingGame.js

```diff
@@ -1,74 +1,73 @@
 function matriz() {
-  let players = document.querySelector(".jugadores");
-  let jugador1 = document.createElement("div");
-  let attJugador1 = document.createAttribute("class");
-  attJugador1.value = "jugador1";
-  jugador1.setAttributeNode(attJugador1);
-  let textoJugador1 = document.createTextNode("JUGADOR X");
-  jugador1.appendChild(textoJugador1);
-  players.appendChild(jugador1);
+  let players = document.querySelector(".players");
+  let player1 = document.createElement("div");
+  let attplayer1 = document.createAttribute("class");
+  attplayer1.value = "player1";
+  player1.setAttributeNode(attplayer1);
+  let textPlayer1 = document.createTextNode("JUGADOR X");
+  player1.appendChild(textPlayer1);
+  players.appendChild(player1);
 
-  let jugador2 = document.createElement("div");
-  let attjugador2 = document.createAttribute("class");
-  attjugador2.value = "jugador2";
-  jugador2.setAttributeNode(attjugador2);
-  let textojugador2 = document.createTextNode("JUGADOR O");
-  jugador2.appendChild(textojugador2);
-  players.appendChild(jugador2);
+  let player2 = document.createElement("div");
+  let attplayer2 = document.createAttribute("class");
+  attplayer2.value = "player2";
+  player2.setAttributeNode(attplayer2);
+  let textPlayer2 = document.createTextNode("JUGADOR O");
+  player2.appendChild(textPlayer2);
+  players.appendChild(player2);
 
```

Comment:
create a function ( in utils.js file) that creates these two texts. Just call the functions twice and pass each text that is going to be for each player.